### DISCUSSION TITLE
Vendor assets not found if shortName != package name

### DISCUSSION THREAD

**Discussion by datashaman** [spatie/laravel-package-tools#137] [Q&A]
I noticed that the readme suggests that we can publish vendor assets using a variation of this:
```
php artisan vendor:publish --tag="your-package-name-config"
```

The assets are being tagged with the _shortName_ instead of the package name, which is fine if those two are the same. But not if they are different. For example a package named `laravel-translators` has a _shortName_ of `translators`, which means the documented way of publishing the vendor assets doesn't find anything.

This does not work:
```
php artisan vendor:publish --tag="laravel-translators-config"
```

This works:
```
php artisan vendor:publish --tag="translators-config"
```

I think it's a bit weird having a very generic tag of `translators-config` be generated and suggest that the code be changed to match the documentation. PR incoming.

**Comment by francoism90** [comment]
@datashaman You can add the correct name in the methods, isn't that the same?
